cpp/benchmarks/CMakeLists.txt

```diff
@@ -42,7 +42,16 @@ function(kvikio_add_benchmark)
   endif()
 
   add_executable(${_KVIKIO_NAME} ${_KVIKIO_SOURCES})
-  set_target_properties(${_KVIKIO_NAME} PROPERTIES INSTALL_RPATH "\$ORIGIN/../../../lib")
+  set_target_properties(
+    ${_KVIKIO_NAME}
+    PROPERTIES INSTALL_RPATH "\$ORIGIN/../../../lib"
+               CXX_STANDARD 20
+               CXX_STANDARD_REQUIRED ON
+               # For std:: support of __int128_t. Can be removed once using cuda::std
```

Comment:
What actions do we need to take here? Where does kvikio use `__int128_t`? I think we want to move to `cuda::std` or remove the `CXX_EXTENSIONS ON` if not needed.